cfme/tests/configure/test_access_control.py

```diff
@@ -85,6 +85,24 @@ def _check_item_visibility(item, user_restricted):
     return _check_item_visibility
 
 
+@pytest.fixture(params=['tag', 'tag_expression'])
+def tag_value(appliance, category, tag, request):
+    tag_type = request.param
+    if tag_type == 'tag':
+        tag_for_create = ([category.display_name, tag.display_name], True)
+        tag_for_update = ([category.display_name, tag.display_name], False)
+    else:
+        if appliance.version > '5.9':
+            if BZ(1579867, forced_streams=['5.9']).blocks:
+                tag_for_create = 'fill_tag(My Company Tags : Cost Center, Cost Center 001)'
+            else:
+                tag_for_create = 'fill_tag(My Company Tags : {})'.format(category.display_name)
+            tag_for_update = 'delete_whole_expression'
+        else:
+            pytest.skip("Tag expression is suported from 5.9 version")
```

Comment:
This should be an uncollect marker applied to the test case, not a skip, which shows up as a failure in 58z.